libevmasm/GasMeter.cpp

```diff
@@ -184,7 +184,7 @@ GasMeter::GasConsumption GasMeter::estimateMax(AssemblyItem const& _item, bool _
 				if (*value)
 				{
 					// Note: msb() counts from 0 and throws on 0 as input.
-					unsigned const significantByteCount  = (boost::multiprecision::msb(*value) + 1 + 7) / 8;
+					unsigned const significantByteCount  = (unsigned)((boost::multiprecision::msb(*value) + 1 + 7) / 8);
```

Comment:
```suggestion
					unsigned const significantByteCount  = static_cast<unsigned>((boost::multiprecision::msb(*value) + 1 + 7) / 8);
```

But I think it would be cleaner to just change the type of `significantByteCount` to `unsigned long`:
```suggestion
					unsigned long const significantByteCount = (boost::multiprecision::msb(*value) + 1 + 7) / 8;
```